backoffice/urls.py

```diff
@@ -5,6 +5,11 @@
     path("", views.home, name="backoffice_home"),
     path("bookings/", views.bookings, name="backoffice_bookings"),
     path("users/", views.users, name="backoffice_users"),
+    path("users/card/add/<int:id>/", views.add_card, name="backoffice_add_card"),
```

Comment:
```suggestion
    path("users/<int:id>/cards/add/", views.add_card, name="backoffice_add_card"),
```
Suggesting this as a potentially cleaner path?